content/800-data-platform/05-data-browser.mdx

```diff
@@ -154,18 +164,17 @@ Use the **Filters** menu to filter data in the model by adding conditions.
       - **gt**
       - **gte**
       - **not**
-   3. Type the value you want to use for the filter.<br />
-      **Step result**: The model is filtered immediately after you define the first filter.
+   3. Type the value you want to use for the condition.<br />
+      **Step result**: The data in the model is updated immediately based on the condition.
 3. To add a new filter, click **Add a new filter** and repeat the steps above.
 4. To remove a filter, click the **x** button on the right.
-   <img src="https://res.cloudinary.com/prismaio/image/upload/v1658948754/docs/pdp/data-browser-filters-actions-v03_fuewwd.png" width="650px" alt="Data Browser - add and remove apply filters"/>
+   <img src="https://res.cloudinary.com/prismaio/image/upload/v1661864213/docs/pdp/data-browser-filters-actions-v04_r0jnps.png" width="650px" alt="Data Browser - add and remove apply filters"/>
 5. To remove all filters, click **Clear all**.
-6. If you have too many filters and want to clear the UI real estate, click **Show less** and only the topmost filter remains visible. Click **Show more** to show again the rest of the applied filters.
 
 **Result**
 
-* The data in the model is filtered based on the conditions you added.
-* The default **None** value in the **Filters** menu changes to the number of filters you added.
+* The data in the model is filtered based on the combination of all conditions you add.
+* The default **None** value in the **Filters** menu changes to the number of filters you add.
```

Comment:
Maybe: "The default value of  **None** changes to display the total number of filters you add."

Reply:
Thank you. Changed now.